config/default.yml

```diff
@@ -4,6 +4,10 @@ AllCops:
     Patterns:
     - _spec.rb
     - "(?:^|/)spec/"
+  FactoryGirl:
```

Comment:
Regarding prefix: yes
Regarding the option, haven't really thought about this. Guess disabling the requires is easier. @backus any thoughts?